tools/codesign_macOS_executable/sign_executable.py

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Please refer to https://medium.com/@parthnaik92/codesign-and-notarize-unreal-engine-mac-builds-for-distribution-outside-of-the-mac-store-d2f6e444f3a7
+# for pre-requisites and setup your system before trying to run this file.
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import time
+
+
+if __name__ == "__main__":
+
+    assert sys.platform == "darwin"
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--executable", required=True)
+    parser.add_argument("--developer_id", required=True)
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--tmp_dir", required=True)
+    parser.add_argument("--apple_username", required=True)
+    parser.add_argument("--apple_password", required=True)
+    parser.add_argument("--entitlements_file", default=os.path.join(os.path.dirname(os.path.realpath(__file__)), "entitlements.plist"))
+    parser.add_argument("--wait_time_seconds", type=float, default=600.0)
+    parser.add_argument("--request_uuid")
```

Comment:
- make it so the default behavior is to do this
- maybe make the parameter called "skip_request_uuid" and use action="store_true" to make it a bool

Reply:
Done